main/SQLHelper.cpp

```diff
@@ -2218,6 +2218,18 @@ bool CSQLHelper::OpenDatabase()
 				}
 			}
 		}
+                if (dbversion < 114)
+                {
+                        //Set default values for new parameters in EcoDevices and Teleinfo EDF
+                        std::stringstream szQuery1, szQuery2, szQuery3;
+			szQuery1 << "UPDATE Hardware SET Mode1 = 0, Mode2 = 60 WHERE Type =" << HTYPE_ECODEVICES ;
+                        query(szQuery1.str());
+                        szQuery2 << "UPDATE Hardware SET Mode1 = 0, Mode2 = 0, Mode3 = 60 WHERE Type =" << HTYPE_TeleinfoMeter ;
+                        query(szQuery2.str());
+			szQuery3 << "UPDATE Hardware SET DataTimeout = DataTimeout + 5 WHERE DataTimeout >0";
```

Comment:
I'm not sure I understand your comment. The "where  >0" clause is there to leave zeroes untouched and watchdogs disabled. Without it the update would replace 0 with a 5 sec timeout, thus enabling the watchdog. All others are increased by 5 sec in order to match the value set by the GUI